src/sass/layout/scanning-page/_hero.scss

```diff
@@ -0,0 +1,37 @@
+.container {
+    width: 320px;
+    padding: 0 15px;
+    margin: 0 auto;
+  }
+.scanning-page {
+    padding-top: 11.35px;
+    padding-bottom: 23.58px;
+}
+.scanning-page__item:not(:last-child) {
+    margin-bottom: 24px;
+}
+
+.scanning-page__title {
+    margin-bottom: 16px;
+
+}
+.scanning-page__desc {
+    font-family: Cera Pro;
+font-style: normal;
+font-weight: 300;
+font-size: 16px;
+line-height: 20px;
+
+color: #2B2B2B;
+}
+
+.scanning-page__title {
```

Comment:
we also have partial of tittles, you can choice what you need and add the class name in your tag